selfdrive/test/process_replay/process_replay.py

```diff
@@ -29,175 +24,165 @@
 PROC_REPLAY_DIR = os.path.dirname(os.path.abspath(__file__))
 FAKEDATA = os.path.join(PROC_REPLAY_DIR, "fakedata/")
 
+
+class ReplayContext:
+  def __init__(self, cfg):
+    self.non_polled_pubs = list(set(cfg.pubs) - set(cfg.polled_pubs))
+    self.polled_pubs = cfg.polled_pubs
+    self.drained_pub = cfg.drained_pub
+    assert(len(self.non_polled_pubs) != 0 or len(self.polled_pubs) != 0 or self.drained_pub is not None)
+  
+  def __enter__(self):
+    messaging.toggle_fake_events(True)
+
+    if len(self.polled_pubs) > 0 and self.drained_pub is None:
+      self.poll_called_event = messaging.fake_event("", messaging.FAKE_EVENT_POLL_CALLED)
+      self.poll_ready_event = messaging.fake_event("", messaging.FAKE_EVENT_POLL_READY)
+      self.recv_called_events = {}
+      self.recv_ready_events = {}
```

Comment:
do we need to handle polled sockets differently? shouldn't hooking recv be enough?

Reply:
with the new FakePoller implementation, these are not needed anymore